shared/domains/chelonia/internals.js

```diff
@@ -1630,6 +1702,9 @@ export default (sbp('sbp/selectors/register', {
         }
         // we revert any changes to the contract state that occurred, ignoring this mutation
         console.warn(`[chelonia] Error processing ${message.description()}: ${message.serialize()}. Any side effects will be skipped!`)
+        if (this.config.strictProcessing) {
+          throw e
+        }
```

Comment:
I'm not sure this is correct.

Users can write contracts that throw errors for whatever reason (intentionally or unintentionally) due to a bug in the contract code, or on purpose. In such cases the codebase has been designed to ignore the mutation, but accept and store the message.

If the server throws however, then what that means is that there are situations where the chain could potentially become unwriteable for an extended period of time. I'm not sure we want to allow this...

What are your thoughts?